cowprotocol/accounting/rewards/mainnet/service_fee_query_4017925.sql

```diff
@@ -0,0 +1,253 @@
+WITH
+
+bonding_pools (name, pool, initial_funder) AS (
+    SELECT
+        name,
+        from_hex(pool),
+        from_hex(funder)
+    FROM (
+        VALUES {{BondingPoolData}}
+    ) AS _ (name, pool,funder)
+),
```

Comment:
I think we can fix the linting errors and also set the bonding pools as constants if we hardcode them like this:

```
WITH
  bonding_pools (pool, name, initial_funder) AS (
    SELECT
      from_hex('0x8353713b6D2F728Ed763a04B886B16aAD2b16eBD') AS pool,
      'Gnosis' AS name,
      from_hex('0x6c642cafcbd9d8383250bb25f67ae409147f78b2') AS initial_funder
    UNION ALL
    SELECT
      from_hex('0x5d4020b9261F01B6f8a45db929704b0Ad6F5e9E6') AS pool,
      'CoW Services' AS name,
      from_hex('0x423cec87f19f0778f549846e0801ee267a917935') AS initial_funder
  ),
```